src/mol-plugin-state/formats/registry.ts

```diff
@@ -53,11 +53,11 @@ export class DataFormatRegistry {
     }
 
     constructor() {
+        for (const [id, p] of BuiltInTrajectoryFormats) this.add(id, p);
```

Comment:
any particular reason to change the order? Is it important? If so, make a note.

Reply:
The .lammpstrj format should be considered as a Trajectory when drag/drop or File Open, but as a Coordinates when using the Load Trajectory button that expect a topology (here .dat) and a trajectory (.lammpstrj). So because .lammpstrj  is in both category, I reorder them so that it will first check for the BuiltInTrajectoryFormats rather than the BuiltInCoordinatesFormats category. For instance, I should be able to load NPC-NUP98_Capsid.data + Nup1.lammpstrj with Load Trajectory, but also simply dropping Nup1.lammpstrj directly. Does that make sense ? Do you think there is another way to approach this? Also the Load Trajectory doesn't work currently, so I need to fix it/

Reply:
I'll need some time to think this through. As it is, it would definitely need a comment so the next person is not just changing it...

Reply:
I revert to the default order so I can use the Load Trajectory UI, and it follow the behavior of the other format. Maybe it could be possible to popup a small widget after the drop to pick trajectory or coordinate provider ?

Reply:
Good call to move this issue out of the PR.